opencensus/trace/internal/span.cc

```diff
@@ -224,6 +224,8 @@ const SpanContext& Span::context() const { return context_; }
 
 bool Span::IsSampled() const { return context_.trace_options().IsSampled(); }
 
+void Span::StopRecording() { span_impl_ = nullptr; }
```

Comment:
This approach will not work, it prevents Span::End from doing:
```
exporter::RunningSpanStoreImpl::Get()->RemoveSpan(span_impl_);
```
(i.e. leaks memory)

Reply:
Ouch. I was not aware of the mechanics of memory management here

Reply:
I'm sorry. I wish things were simple.